app/working-with-app.md

```diff
@@ -372,57 +372,60 @@ port `8080` is allowed on the connection form your browser to your Docker host.
 
 ### Push the app to Docker Hub
 
-As mentioned in the introduction, `docker app` lets you manage entire applications the same way that you 
-currently manage container images. For example, you can push and pull entire applications from registries like 
-Docker Hub with `docker app push` and `docker app pull`. Other `docker app` commands, such 
-as `install`, `upgrade`, `inspect` and `render` can be performed directly on applications while they are 
-stored in a registry.
+As mentioned in the introduction, `docker app` lets you manage entire
+applications the same way that you currently manage container images. For
+example, you can push and pull entire applications from registries like Docker
+Hub with `docker app push` and `docker app pull`. Other `docker app` commands,
+such as `install`, `upgrade`, `inspect` and `render` can be performed directly
```

Comment:
`inspect`, and `render`

Reply:
Fixed in https://github.com/docker/docker.github.io/pull/9170